docker/start.sh

```diff
@@ -119,16 +120,29 @@ wait_for_postgres() {
 }
 
 
-wait_for_es() {
+setup_es() {
+    SECONDS=0
+    duration=0
```

Comment:
we probably don't need this line, but it doesn't hurt either.

Reply:
resolved